src/agent.rs

```diff
@@ -65,13 +60,12 @@ impl Agent {
                 command.arg(min.to_string());
             }
             if let Some(ref cipher) = a.cipher {
+                if check_cipher_blacklist(cipher, path) {
+                    return Err(89);
```

Comment:
Can these errors get names? No idea what `89` is (well sort of :wink: ).

Reply:
I named this specific error. It's the same error code that is returned by a shim if some flag is not implemented, and it results in tests being counted as skipped instead of failed. 
I'm not very familiar with how and where to define consistent names for error codes, so I just gave it a try.